src/scenes/ProgressReports/EditForm/InstructionsDialog.tsx

```diff
@@ -20,31 +20,75 @@ export const InstructionsDialog = ({ open, onClose }: DialogProps) => {
+          inform Investors. Each section here currently informs our investors on 
+          what the team has been doing, how God’s Word is impacting the lives of 
+          the language community, and the language progress.
         </Typography>
         <Typography sx={{ mb: 2 }}>
-          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Elit pharetra
-          enim justo, molestie amet viverra faucibus. Egestas congue felis arcu.
+          Please ensure ALL information is scrubbed for security & sensitivity.
         </Typography>
-        <Typography sx={{ mb: 2 }}>Sensitivity Check list</Typography>
         <Typography sx={{ mb: 2 }}>
-          Please check over your report to make sure there are no security
-          concerns.
+          After reviewing and filling out all sections for your role, submit or 
+          approve the Quarterly Report under Final Details.
         </Typography>
         <Typography sx={{ mb: 2 }}>
-          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Elit pharetra
-          enim justo, molestie amet viverra faucibus. Egestas congue felis arcu.
+          Sensitivity Checklist
         </Typography>
         <Typography sx={{ mb: 2 }}>
-          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Elit pharetra
-          enim justo, molestie amet viverra faucibus. Egestas congue felis arcu.
+          - Replace Personal and Place Names with Pseudonyms
+          - Remove contact information
+          - Remove or make generic any other identifying information
         </Typography>
         <Typography sx={{ mb: 2 }}>
-          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Elit pharetra
-          enim justo, molestie amet viverra faucibus. Egestas congue felis arcu.
+          Section Instructions for Field Project Managers
+        </Typography>
+        <Typography sx={{ mb: 2 }}>
+          Team News
+            - Please write a succinct reflection of the translation activities 
+              and what the team has been doing over the last quarter. 
+            - Write as if you were speaking to an Investor. 
+            - The communications team will edit grammar prior to publishing and 
+              sending to the Investor.  
+        </Typography>
+        <Typography sx={{ mb: 2 }}>
+          Community Story
+            - Review and scrub the Field Partner and Translation variants.
+            - Please provide any additional context, correction, or clarity on 
+              the story submitted by the Field Partner - write out acronyms, 
+              clarify names & their role with the project, and add context that 
+              was assumed in the Partners submission.
+            - This information will provide context for the Communications Writer 
+              to write and edit the story.
+            - No input is needed if the story is clear in the Field Partner or 
+              Translation variant.
```

Comment:
@sethmcknight We do not assume nothing means "take from previous". If the story is clear the expectation is they copy it into the FPM variant. Is that ok?